hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -1717,7 +1717,7 @@
         Full-flush invalidation means the invalidation of events for all entries when clear is called.
         The default value is false.
     -->
-    <cache name="default">
```

Comment:
This should basically show the users the following
- all elements allowed
- what's the default value for those elements, I think the `default` name comes from there, the values should be the same as when you do `new CacheConfig()`. The value could be as well `sample-full-cache-name` or anything else, wdyt?

These are examples, so changing this doesn't have any effect, If you think it clarifies things then go for it, but it should be updated everywhere:, e.g. `<multimap name="default">` above, also `<replicatedmap name="replicatedMap1">`